certbot/main.py

```diff
@@ -652,15 +652,18 @@ def renew_cert(config, plugins, lineage):
 
     notify = zope.component.getUtility(interfaces.IDisplay).notification
     if installer is None:
-        notify("new certificate deployed without reload, fullchain is {0}".format(
-               lineage.fullchain), pause=False)
+        if not config.quiet:
```

Comment:
I'm sorry I missed this earlier, but these `if not config.quiet` lines here and below aren't needed because the notification method is essentially a noop if `config.quiet` is set.

Reply:
Revert changes.